database_initializer.py

```diff
@@ -44,16 +50,19 @@ def execute(self):
         self._populate_playlist_table()
         self._populate_artist_table()
         self._populate_track_table()
+        self._populate_album_table()
         self._clean_temp_folder()
-    
+        
     def _clean_temp_folder(self):
         for file in os.scandir('temp'):
             os.remove(file)
 
 
 if __name__ == '__main__':
-    if len(sys.argv) == 3:
+    if len(sys.argv) == 5:
```

Comment:
This check will fail if the program is run with only two cli agruments. Hence, it will never be executed. Please fix this.